spyder_terminal/config.py

```diff
@@ -20,6 +20,13 @@
       'shell': 'cmd' if WINDOWS else 'bash'
      }
      ),
+    ('shortcuts',
```

Comment:
Please add a shortcut to clear the terminal, which is a pretty common operation too.